.pre-commit-config.yaml

```diff
@@ -11,6 +11,7 @@ repos:
       - id: check-json
       - id: check-toml
       - id: check-yaml
+        exclude: ^\.gitlab-ci.yml
```

Comment:
Why does the gitlab-ci.yml need to be excluded?

Reply:
@a-collet : seems ok to have .gitlab-ci.yml reformated. Mind I remove this line? 
thank you!

Reply:
I believe it was breaking some syntax and that's why I excluded it. Feel free to remove the exclude and try again.